### ISSUE TITLE
"Subscribe" button in newsletter part of footer has insufficient colour contrast

### ISSUE THREAD

**Issue by tremby** [tremby/markwiens.ca-ubertor#21]
The white on yellow is borderline impossible to read; many users will have trouble reading it.

![2023-08-23_18-56](https://github.com/tremby/markwiens.ca-ubertor/assets/199635/7b707d1b-9a6e-46ad-96d4-8037dcd2830e)
